### PR TITLE
Object storage

### PR DESCRIPTION
**Pull request by sagikazarmark**
SplObjectStorage is proven to be faster than arrays.

This PR adds an extended DateTimeStorage which calculates the hash and stores objects.

**Note:** This requires #8 to be merged

### CODE REVIEW

File: src/DateTimeStorage.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * (c) Florian Voutzinos <florian@voutzinos.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Business;
+
+/**
+ * Default implementation of BusinessInterface.
```
Comment (florianv) [inline src/DateTimeStorage.php]: Wrong comment

File: src/DateTimeStorage.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ *
+ * @author Márk Sági-Kazár <mark.sagikazar@gmail.com>
+ */
+final class DateTimeStorage extends \SplObjectStorage
+{
+    public function getHash($object)
```
Comment (florianv) [inline src/DateTimeStorage.php]: Please add the @inheritdoc annotation

File: src/DateTimeStorage.php (hunk 1 above)
Comment (sagikazarmark) [inline src/DateTimeStorage.php]: Cause by not rebased #8

**Comment by sagikazarmark** [comment]
Branch rebased. Please check it. If OK, I squash the commits.

File: src/DateTimeStorage.php (hunk 1 above)
Comment (florianv) [inline src/DateTimeStorage.php]: No

Comment (sagikazarmark) [inline src/DateTimeStorage.php]: Indeed, fixed

**Comment by florianv** [comment]
Looks good to merge :)

**Comment by sagikazarmark** [comment]
Squashing...

**Comment by sagikazarmark** [comment]
Squashed

**Comment by florianv** [comment]
Thanks!

**Comment by sagikazarmark** [comment]
My pleasure, great library :+1:, worth the effort.

**Pull request merged**